.vib/wordpress/goss/wordpress.yaml

```
@@ -1,6 +1,6 @@
 command:
   check-app-version:
-    exec: cat /opt/bitnami/wordpress/wp-includes/version.php  | grep "wp_version"
+    exec: sed -nE "s/wp_version\s*=\s*'([0-9\.]+)';/\1/p" /opt/bitnami/wordpress/wp-includes/version.php | tr -d $ | sed -E 's/^[0-9]+\.[0-9]+$/&.0/'
     exit-status: 0
     stdout:
       - {{ .Env.APP_VERSION }}
```

We have faced something similar on other assets like [golang](https://github.com/bitnami/containers/blob/20c6ccdc63841242bd75ffccf2d2123150fb8a9e/.vib/golang/goss/golang.yaml#L3)

```suggestion
    exec: cat /opt/bitnami/wordpress/wp-includes/version.php | grep "wp_version" | grep "$(echo $APP_VERSION | sed -E 's|([0-9]+\.[0-9]+)\.0$|\1|g')"
    exit-status: 0
```

This way, the change is only applied when the version is a new Major or Minor (`X.0.0` or `Y.0.0`)

---

In terms of readability, I personally prefer the suggested version. But it's okay.